editor/editor_properties.cpp

```diff
@@ -128,6 +128,18 @@ void EditorPropertyText::set_placeholder(const String &p_string) {
 	text->set_placeholder(p_string);
 }
 
+void EditorPropertyText::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE:
```

Comment:
You don't need `NOTIFICATION_ENTER_TREE` btw. `NOTIFICATION_THEME_CHANGED` is sent automatically during that phase.

Reply:
Removed. Thanks!